src/components/pages/AccountSettings/RenderAccountSettings.js

```diff
@@ -52,7 +50,7 @@ function RenderAccountSettings() {
         onOk={unlock ? null : handleOk}
         okType="default"
         onCancel={handleCancel}
-        afterClose={form.resetFields()}
+        afterClose={() => pin.clear()}
         centered="true"
```

Comment:
pin.clear is how react-pin-input handles clearing inputs as opposed to antD's form.resetFields()